free-1.md

```diff
@@ -1,50 +1,156 @@
 # Free project 1
 
-## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+Calendar DSL
 
+## The user and a language
+This section describes who the project would serve and why a language might be
+a good way to meet their needs.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A calendar DSL would help people schedule events in a more efficient and less
+error-prone manner.  There are many times when people must schedule lots of
+events on a calendar end up performing many repetitive tasks.  For example, at
+the beginning of the school year, students often input all of their classes into
+a Google Calendar.  Many of these classes have similar traits, but as far as I
+know, using the Google Calendar App on my phone, there is no easy way of adding
+many different types of the same event.  Therefore, if you want to, for example,
+set a notification 5 minutes before your class started, you would have to
+manually change the settings from the default version for each class.  With a
+DSL, students and professionals alike would be able to add events in a much more
+intuitive and efficient way.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Some sort of automation would be helpful to expedite the process of adding
```

Comment:
You might be interested in a project from last year

https://github.com/hmc-cs111-fall2015/project-ideas/pull/7
https://github.com/hmc-cs111-fall2015/project-pitch/pull/6
https://github.com/hmc-cs111-fall2015/project/pull/4/files

iirc he ended up with a pretty clean solution.